source/server/http/admin.h

```diff
@@ -304,6 +305,7 @@ class AdminImpl : public Admin,
   Http::ConnectionManagerTracingStats tracing_stats_;
   NullRouteConfigProvider route_config_provider_;
   std::list<UrlHandler> handlers_;
+  int32_t max_request_headers_size_kb_{Http::DEFAULT_MAX_REQUEST_HEADERS_SIZE_KB};
```

Comment:
Why is it a `uint32_t` everywhere but here?

Reply:
oops